### PR TITLE
[MonologBridge] Bump min version for monolog ^1.25 and drop dead code

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Comment by lyrixx** [comment]
I block this PR: I will need https://github.com/Seldaek/monolog/pull/1374 and a new release

File: composer.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
         "doctrine/reflection": "~1.0",
         "doctrine/doctrine-bundle": "~1.4",
         "masterminds/html5": "^2.6",
-        "monolog/monolog": "~1.11",
+        "monolog/monolog": "~1.25",
```
Comment (nicolas-grekas) [inline composer.json]: `^1.25.1`

File: src/Symfony/Bridge/Monolog/composer.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "monolog/monolog": "~1.19",
+        "monolog/monolog": "^1.25",
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Monolog/composer.json]: ^1.25.1

**Comment by lyrixx** [comment]
Travis failures are not related

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Comment by derrabus** [comment]
Does that change also affect the minimal Monolog 2 version on master?

**Comment by lyrixx** [comment]
@derrabus I think it should be ok

**Pull request merged**